native_client/java/Makefile

```diff
@@ -1,30 +0,0 @@
-.PHONY: clean apk-clean
```

Comment:
Removing that file means you basically broke everything

Reply:
What exactly?

Reply:
Everything, all our build system.